Dozer/DozerIcons.swift

```diff
@@ -255,9 +278,20 @@ public final class DozerIcons {
         }
     }
 
-    /// Determines if the user is interacting with the status bar based on level, owner and y-coordinate
+    /// hide and show dock icon and thus its menu bar
```

Comment:
Personally I would put in *why* the function does that, i.e. “so the menu bar is freed up and all icons are visible”.

Reply:
have done so! cheers